lib/src/resultsetTable/ResultsetTable.stories.tsx

```diff
@@ -0,0 +1,276 @@
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Small" theme="light" level={4} />
+      <DxcResultsetTable columns={columns} rows={rows} margin={"small"}></DxcResultsetTable>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Medium" theme="light" level={4} />
+      <DxcResultsetTable columns={columns} rows={rows} margin={"medium"}></DxcResultsetTable>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Large" theme="light" level={4} />
+      <DxcResultsetTable columns={columns} rows={rows} margin={"large"}></DxcResultsetTable>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xlarge" theme="light" level={4} />
+      <DxcResultsetTable columns={columns} rows={rows} margin={"xlarge"}></DxcResultsetTable>
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xxlarge" theme="light" level={4} />
+      <DxcResultsetTable columns={columns} rows={rows} margin={"xxlarge"}></DxcResultsetTable>
+    </ExampleContainer>
+  </>
+);
+
+const ResultsetTableAsc = () => (
+  <ExampleContainer>
+    <Title title="Ascendant sorting" theme="light" level={4} />
+    <DxcResultsetTable columns={columnsSortable} rows={rowsSortable}></DxcResultsetTable>
+  </ExampleContainer>
+);
+
+export const AscendentSorting = ResultsetTableAsc.bind({});
+AscendentSorting.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  await userEvent.click(canvas.queryByText("Name"));
+};
+
+const ResultsetTableDesc = () => (
+  <ExampleContainer>
+    <Title title="Descendant sorting" theme="light" level={4} />
+    <DxcResultsetTable columns={columnsSortable} rows={rowsSortable}></DxcResultsetTable>
+  </ExampleContainer>
+);
+
+export const DescendantSorting = ResultsetTableDesc.bind({});
+DescendantSorting.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  await userEvent.click(canvas.queryByText("Name"));
+  await userEvent.click(canvas.queryByText("Name"));
+};
+
+const ResultsetTableMiddle = () => (
+  <ExampleContainer>
+    <Title title="Middle page" theme="light" level={4} />
+    <DxcResultsetTable columns={columns} rows={rows} itemsPerPage={2}></DxcResultsetTable>
+  </ExampleContainer>
+);
+
+export const MiddlePage = ResultsetTableMiddle.bind({});
+MiddlePage.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const nextButton = canvas.getAllByRole("button")[2];
+  await userEvent.click(nextButton);
+};
+
+const ResultsetTableLast = () => (
+  <ExampleContainer>
+    <Title title="Last page" theme="light" level={4} />
+    <DxcResultsetTable columns={columns} rows={rows} itemsPerPage={2}></DxcResultsetTable>
+  </ExampleContainer>
+);
+
+export const LastPage = ResultsetTableLast.bind({});
+LastPage.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+  const nextButton = canvas.getAllByRole("button")[3];
+  await userEvent.click(nextButton);
+};
```

Comment:
This test is not necessary as this case is already tested in the paginator visual tests.

Reply:
This test is added to check functionality of the last button, to see the table state in the last page.

Reply:
Got it! 👍🏻